Informal List of Projects

```diff
@@ -0,0 +1,66 @@
+###Code Repository
+A kind of Gitbuh but decentralised. Rewards based on commits.
+
+###Security Cameras 
+Use case: you can install security cameras in a kindergarten but no-one can access the footage unless 51% of parents give the consent to do it. It would solve the privacy vs security issue.
+
+###Plugin as Coinpayments
+But open, as with myetherwallet, you can add custom token just defining name, address and decimals
+
+###Call-Center / Customer-Care on demand
+Uber economy; Decentralised customer care on demand.
+It would be also possible to create services for marketplaces (i.e. themeforest) where support is based on some reward system
+
```

Comment:
Love the idea, would be important to understand how we can enforce quality.

Reply:
Maybe in the same way freelance platforms do: with a rating system